backend/src/modules/tournament-tables/core/domain/event/TableWasIncludedInAvailableTables.ts

```diff
@@ -0,0 +1,12 @@
+import { DomainEvent } from '../../../../../shared/domain/event/DomainEvent';
+import { TournamentTable } from '../TournamentTable';
+
+export class TableWasIncludedInAvailableTables implements DomainEvent {
+  readonly occurredAt: Date;
+  readonly tableIncluded: TournamentTable;
```

Comment:
Tutaj dobrze byłoby dać tylko tournamentId i tableNumber, bo to opisuje event. Teraz mamy obiekt, który ma np. property availableToPlay, który to jest nadmiarowy, samo przez sie się rozumie patrząc na typ eventu.